src/test/java/io/strimzi/kafka/bridge/http/StaticTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.http;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class StaticTest {
```

Comment:
out of curiosity, why this name?

Reply:
I set this name as StaticTest becasue I am setting the kafka bridge up and down using the the bridge jar file, not using the test classes. This Static Classes does not extends any test class, like HttpBridgeTestBase for instance

Reply:
So this class expects a bridge already running not deploying one by itself. I have no great names in mind, at least please add some JavaDoc on the class.

Reply:
actually no, what I meant is that I am setting it up and down using the ProcessBuilder.
So I had to skip all the @afterall, @aftereach and @beforeall.
I will add the javadocs